exercises/sgf-parsing/canonical-data.json

```
@@ -170,6 +170,185 @@
+            },
+            "expected": {
+                "properties": {
+                    "A": ["x[y]z", "foo"],
+                    "B": ["bar"]
+                },
+                "children": [
+                    {
+                        "properties": {
+                            "C": ["baz"]
+                        },
+                        "children": []
+                    }
+                ]
+            }
+        },
+        {
+            "uuid": "a780b97e-8dbb-474e-8f7e-4031902190e8",
+            "description": "Semicolon in property value doesn't need to be escaped",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[a;b][foo]B[bar];C[baz])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["a;b", "foo"],
+                    "B": ["bar"]
+                },
+                "children": [
+                    {
+                        "properties": {
+                            "C": ["baz"]
+                        },
+                        "children": []
+                    }
+                ]
+            }
+        },
+        {
+            "uuid": "0b57a79e-8d89-49e5-82b6-2eaaa6b88ed7",
+            "description": "Parentheses in property value don't need to be escaped",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[x(y)z][foo]B[bar];C[baz])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["x(y)z", "foo"],
+                    "B": ["bar"]
+                },
+                "children": [
+                    {
+                        "properties": {
+                            "C": ["baz"]
+                        },
+                        "children": []
+                    }
+                ]
+            }
+        },
+        {
+            "uuid": "c72a33af-9e04-4cc5-9890-1b92262813ac",
+            "description": "Escaped tab in property value is converted to space",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[hello\\\tworld])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["hello world"]
+                },
+                "children": []
+            }
+        },
+        {
+            "uuid": "3a1023d2-7484-4498-8d73-3666bb386e81",
+            "description": "Escaped newline in property value is converted to nothing at all",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[hello\\\nworld])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["helloworld"]
+                },
+                "children": []
+            }
+        },
+        {
+            "uuid": "25abf1a4-5205-46f1-8c72-53273b94d009",
+            "description": "Escaped t and n in property value are just letters, not whitespace",
```

For consistency with the other test cases:

```suggestion
            "description": "escaped t and n in property value are just letters, not whitespace",
```